distribution/examples/monitoring-tracing/opentelemetry/README.md

```
@@ -1,98 +1,59 @@
-      </request>
-      <response>
-         <template>Hello from a faked backend!</template>
-      </response>
-      <return/>
-   </api>
-</router>
+A span created by Membrane should be visible in the [Jaeger UI](http://localhost:16686).
+![sample](resources/otel_example.png)
+7. Check the headers printed to the console. You will see headers such as `traceparent`, which indicate the trace and span context involved in the request.
+
+**How it is done**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)